firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift

```diff
@@ -282,85 +289,86 @@ class SearchTests: BaseTestCase {
         waitForTabsButton()
         validateSearchSuggestionText(typeText: "localhost")
     }
+
 // TODO: Add UI Tests back when felt privay simplified UI feature flag is enabled or when we support feature flags for tests
 //    func testPrivateModeSearchSuggestsOnOffAndGeneralSearchSuggestsOn() {
 //        navigator.nowAt(NewTabScreen)
 //        navigator.goto(SearchSettings)
 //        navigator.nowAt(SearchSettings)
-//
+//        
```

Comment:
You may need to remove these extra spaces here

Reply:
Added the tests back with the exact spacing as before.